docs/amber_script.md

```diff
@@ -218,26 +223,55 @@ value for `START_IDX` is 0. The default value for `COUNT` is the item count of
 vertex buffer minus the `START_IDX`.
 
 ```
+// Run the given |pipeline_name| which must be a `compute` pipeline. The
+// pipeline will be run with the given |x|, |y|, |z| values.
```

Comment:
VkScript isn't clear about what these are.  
We should match vkCmdDispatch, in which case these are the number of workgroups to run in the x,y, and z dimensions.

Reply:
Done.